xc7/tests/buttons/CMakeLists.txt

```diff
@@ -1,6 +1,8 @@
+set(BOARDS basys3 zybo)
```

Comment:
This doesn't fix the but with explicit IO files does it?

Reply:
I have actually used the same paradigm as for the Ice40. The fix should be the `IMPLICIT_INPUT_IO_FILES`. Probably I should just remove the scoped variable

Reply:
Sure. So, As far as I got from debugging the should be a bug in the handling of the explicit `INPUT_IO_FILES`. I need to further check it, but for what I saw there is no use of explicit IO files in the repo. Maybe there should be further checks to guarantee that the `INPUT_IO_FILE` list is correctly parsed and passed to the actual `add_fpga_target` function. I will open the ticket right now